src/main/groovy/org/transmartproject/batch/db/PostgresPartitionTasklet.groovy

```diff
@@ -132,6 +147,7 @@ class PostgresPartitionTasklet implements Tasklet {
             partitionId = matcher[0] as int
         }
 
+        //FIXME Where we delete data if table already exists
```

Comment:
It's not clear for me where we delete old records from existing partition tables when we re-upload data. Or we keep creating new partition tables for each upload? See https://github.com/transmart/transmart-data/blob/support/foreign-keys/ddl/postgres/tm_cz/functions/i2b2_process_mrna_data.sql#L1084